http.go

```diff
@@ -0,0 +1,36 @@
+package server
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+)
+
+// writeError writes err to w in JSON format. If err is
+// internal, obfuscates message and logs it the stdout.
+func writeError(w http.ResponseWriter, err error) {
+	code := errorCode(err)
+	msg := errorMessage(err)
+
+	if code >= http.StatusInternalServerError {
+		// Obfuscate message.
+		msg = http.StatusText(code)
+	}
+	// TODO verbose mode and/or logging to file.
+	log.Println(err.Error())
+
+	w.Header().Set("X-Content-Type-Options", "nosniff")
```

Comment:
I found it [here](https://cs.opensource.google/go/go/+/refs/tags/go1.17.7:src/net/http/server.go;l=2056) in the standard lib, for errors only. Googled the thing and stopped myself from going into a rabbit hole (for now...) of what it implies on all requests by only using it with errors too.

😅

Reply:
Mmh, maybe because it's the only function from `http` that writes a response?
Otherwise I don't see why it would be only related to "error responses" (that doesn't even make sense, it's just a response with a given code)